docs/plot_references/plot_reference.rst

```diff
@@ -572,39 +580,36 @@ more spatially correlated the variable.
 The default aggregation function is ``np.mean``, but you can configure the aggregation
 by passing a different function to ``agg``.
 
-.. code-block:: python
+.. plot::
+    :context: close-figs
 
     gplt.quadtree(
         collisions, nmax=1,
-        projection=gcrs.AlbersEqualArea(), clip=boroughs,
-        hue='NUMBER OF PEDESTRIANS INJURED', cmap='Reds', k=None,
+        projection=gcrs.AlbersEqualArea(), clip=boroughs.simplify(0.001),
+        hue='NUMBER OF PEDESTRIANS INJURED', cmap='Reds',
         edgecolor='white', legend=True,
     )
 
-.. image:: ../figures/quadtree/quadtree-k.png
-
 To use a categorical colormap, set ``scheme``.
 
-.. code-block:: python
+.. plot::
+    :context: close-figs
 
     gplt.quadtree(
         collisions, nmax=1,
-        projection=gcrs.AlbersEqualArea(), clip=boroughs,
+        projection=gcrs.AlbersEqualArea(), clip=boroughs.simplify(0.001),
         hue='NUMBER OF PEDESTRIANS INJURED', cmap='Reds', scheme='Quantiles',
         edgecolor='white', legend=True
     )
 
-.. image:: ../figures/quadtree/quadtree-hue.png
-
 Here is a demo of an alternative aggregation function.
 
-.. code-block:: python
+.. plot::
+    :context: close-figs
 
     gplt.quadtree(
         collisions, nmax=1, agg=np.max,
-        projection=gcrs.AlbersEqualArea(), clip=boroughs,
+        projection=gcrs.AlbersEqualArea(), clip=boroughs.simplify(0.001),
```

Comment:
Probably not this PR, but this geometry should be pre-simplified at rest to remove the need for this.